pos_receipt_hide_user/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+{
+    "name": "POS Receipt Hide User",
+    "summary": "Add button to remove user from receipt.",
+    "author": "Tris Doan",
```

Comment:
```suggestion
    "author": "Camptocamp, Odoo Community Association (OCA)",
```

Reply:
- don't worry, you can add yourself in
`readme/CONTRIBUTORS.rst`, in the format `* Tris Doan <tridm@trobz.com>`
- please study a bit about this readme folder in other oca modules
- then create `CREDITS.rst`:
```
The development of this module has been financially supported by:

* Camptocamp
```
- and DESCRIPTION.rst: same content as summary key